packages/hello-world/README.md

```diff
@@ -1 +1,26 @@
 # HelloWorld
+
+Mineral UI `HelloWorld` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+## Installation
+
+```sh
+npm install --save @mineral-ui/hello @mineral-ui/world @mineral-ui/hello-world
+```
+
+## Usage
+
+```jsx
+import React, { Component } from 'react';
+import HelloWorld from '@mineral-ui/hello-world';
```

Comment:
This component actually requires that you import both the hello and world components separately as they are peer dependencies.